Recurly/BaseClient.cs

```diff
@@ -107,6 +108,16 @@ public class BaseClient {
             return resp;
         }
 
+        public void _SetApiUrl(string uri) {
+          if (System.Environment.GetEnvironmentVariable("RECURLY_INSECURE") == "true") {
+            Console.WriteLine("[SECURITY WARNING] _SetApiUrl is for testing only and not supported in production.");
```

Comment:
I just realized that this `Console.WriteLine` is the same in both the `if` and the `else`. We should consider pulling it out of the logic and just print it first, then do the control flow.